ui/__init__.py

```diff
@@ -48,6 +49,8 @@ def draw_add_sketch_in_add_menu(self, context: Context):
 
 
 def register():
+    icon_manager.load_preview_icons()
```

Comment:
Shouldn't this be called from the icon_manager.load() function? Or at least from the same place..

Reply:
Absolutely, but there's a combination of reasons I left it here - icon_manager doesn't have an unload() function so I couldn't unload_preview_icons() there, and it felt asymmetric and inconsistent to have only unload_preview_icons() in `ui/__init__.py` without load (inevitably someone in the future will think it was an accident and will re-add it), so I figured the best course of action was to make load_preview_icons() idempotent and call it in both places. If you think one of the other alternatives above is better, or that I should add an unload() to icon_manager, let me know! I will merge this for now but we can always add more commits to main branch.

Reply:
An unload function in icon_manager.py that is called from the root init file's unregister function seems reasonable to me 👍🏼

Reply:
Done in  8f67f30 :+1: